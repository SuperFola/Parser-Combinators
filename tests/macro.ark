(macro a 1)
(macro b () 2)
(macro
c
(
d
e
)
3)
(macro f(g)4)