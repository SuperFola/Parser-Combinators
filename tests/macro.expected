( Keyword:macro Symbol:a Number:1 )
( Keyword:macro Symbol:b ( ) Number:2 )
( Keyword:macro Symbol:c ( Symbol:d Symbol:e ) Number:3 )
( Keyword:macro Symbol:f ( Symbol:g ) Number:4 )
( Keyword:macro Symbol:h ( Symbol:i Symbol:j ) ( Keyword:let Symbol:a Number:1 ) )
( Keyword:macro Symbol:h ( Symbol:i Symbol:j ) ( Keyword:let Symbol:a ( Keyword:if Symbol:i Number:2 Number:3 ) ) )